### PR TITLE
Expand R intro lectures

### PR DESCRIPTION
**Pull request by ethanwhite**
Make these lectures more step by step

### CODE REVIEW

File: materials/r-intro.md [hunk 1]
```diff
@@ -28,33 +28,116 @@ language: R
...
+  because we typically don't want to look at the details of how numbers are
+  rounded or lists of numbers are sorted.
+
+```
+abs(-2)
+```
+
+* A function call is composed of two parts, the name of the function and the
+  arguments that the function requires to calculate the value it returns. In the
+  example above abs() is the name of the function, and -2 is the argument.
```
Comment (brymz) [inline materials/r-intro.md]: abs() -> `abs()`

Comment (brymz) [inline materials/r-intro.md]: backticks for `-2`, `pi`, `round()`, `3.14159`, `2`... Could you `round(pi, 2)`?

File: materials/data-structures-R.md [hunk 2]
```diff
@@ -72,12 +82,37 @@ surveys <- data.frame(sites, counts, weights)
...
 
-> Do [Exercise 3 - More Variables]({{ site.baseurl }}/exercises/Expressions-and-variables-more-variables-R/).
+
+### Importing data
+
+* `read.csv()`
+* Download the file for the Shrub Volume 2 exercise
+* How do we tell R where this file is
+
+> When someone says `setwd` discuss issues
```
Comment (brymz) [inline materials/data-structures-R.md]: Could you include the `setwd()` issues you hope to discuss? I tend to use the `'Session' -> 'Set Working Directory'` menu option. Perhaps, I should reconsider.

**Comment by ethanwhite** [comment]
Backticks fixed and note about `setwd` issues added.

**Pull request merged**